tasks/eva_2850/fix_discordant_variants.py

```diff
@@ -0,0 +1,337 @@
+    else:
+        logger.info(f"No hash collision for RS {rs_variant['accession']}")
+        correct_rs_start_and_insert(rs_variant, rs_with_new_start)
+
+
+def correct_rs_start_and_insert(rs_variant, rs_with_new_start):
+    dbsnp_cve_collection = mongo_source.mongo_handle[mongo_source.db_name][DBSNP_CLUSTERED_VARIANT_ENTITY]
+
+    drop_statements = []
+    insert_statements = []
+    drop_statements.append(pymongo.DeleteOne({'_id': rs_variant['_id']}))
+    insert_statements.append(pymongo.InsertOne(rs_with_new_start))
+
+    result_drop = dbsnp_cve_collection.bulk_write(requests=drop_statements, ordered=False)
+    logger.info('There was %s old documents dropped' % result_drop.deleted_count)
+    result_insert = dbsnp_cve_collection.bulk_write(requests=insert_statements, ordered=False)
+    logger.info('There was %s new documents inserted' % result_insert.inserted_count)
+
+
+def resolve_collision_and_insert_rs(rs_variant, rs_with_new_start, variant_in_db):
+    dbsnp_cve_collection = mongo_source.mongo_handle[mongo_source.db_name][DBSNP_CLUSTERED_VARIANT_ENTITY]
+    dbsnp_cvoe_collection = mongo_source.mongo_handle[mongo_source.db_name][DBSNP_CLUSTERED_VARIANT_OPERATION_ENTITY]
+
+    dbsnp_cve_drop_statements = []
+    dbsnp_cve_insert_statements = []
+    dbsnp_cvoe_insert_statements = []
+
+    if rs_with_new_start['accession'] < variant_in_db['accession']:
+        dbsnp_cve_drop_statements.append(pymongo.DeleteOne({'_id': rs_variant['_id']}))
+        dbsnp_cve_drop_statements.append(pymongo.DeleteOne({'_id': variant_in_db['_id']}))
+        dbsnp_cve_insert_statements.append(pymongo.InsertOne(rs_with_new_start))
+
+        merge_event = create_merge_event(variant_in_db, rs_with_new_start)
+        dbsnp_cvoe_insert_statements.append(pymongo.InsertOne(merge_event))
+
+        result_drop = dbsnp_cve_collection.bulk_write(requests=dbsnp_cve_drop_statements, ordered=False)
+        logger.info('There was %s old documents dropped' % result_drop.deleted_count)
+        result_cve_insert = dbsnp_cve_collection.bulk_write(requests=dbsnp_cve_insert_statements, ordered=False)
+        logger.info('There was %s new documents inserted' % result_cve_insert.inserted_count)
+        result_cvoe_insert = dbsnp_cvoe_collection.bulk_write(requests=dbsnp_cvoe_insert_statements, ordered=False)
+        logger.info('There was %s new documents inserted' % result_cvoe_insert.inserted_count)
+
+        update_ss_with_new_rs(variant_in_db['accession'], rs_with_new_start['accession'])
+
+    else:
+        dbsnp_cve_drop_statements.append(pymongo.DeleteOne({'_id': rs_variant['_id']}))
+
+        result_drop = dbsnp_cve_collection.bulk_write(requests=dbsnp_cve_drop_statements, ordered=False)
+        logger.info('There was %s old documents dropped' % result_drop.deleted_count)
+
+        update_ss_with_new_rs(rs_with_new_start['accession'], variant_in_db['accession'])
+
+
+def create_merge_event(variant_merged, variant_retained):
+    merge_event = {
```

Comment:
Suggest encoding some useful information into _id (similar to [this](https://github.com/EBIvariation/eva-tasks/pull/75/files#diff-715d5a70989ff39bd36efc22d80598a657cdabd00f439aa6fe1bd818fed3c104R49)) so that we can leverage the index on the _id field.

Reply:
done